packages/core/medusa-test-utils/src/medusa-test-runner.ts

```diff
@@ -172,6 +179,14 @@ export function medusaIntegrationTestRunner({
       throw error
     }
 
+    try {
+      console.log(`Migrating database ${dbName}`)
+      await migrateDatabase()
```

Comment:
Just realized another problem - without the migrations the defaults will not run 😄